### PR TITLE
Geo data frames

### PR DESCRIPTION
**Pull request by mrozpawel**
```
============================================================= test session starts ==============================================================
platform darwin -- Python 3.11.3, pytest-7.3.1, pluggy-1.0.0
rootdir: /Users/pawelmrz/Documents/persistent/nzpyida
plugins: flaky-3.7.0, anyio-3.6.2
collected 479 items                                                                                                                            

test_aggregation.py .................................                                                                                    [  6%]
test_association_rules.py .........                                                                                                      [  8%]
test_base.py .....s........ss..............                                                                                              [ 15%]
test_base_connexion.py ............                                                                                                      [ 17%]
test_base_private.py ......................                                                                                              [ 22%]
test_base_table_manipulation.py ..s..sss..................                                                                               [ 27%]
test_feature_selection.py ssssssssssss..........................................                                                         [ 38%]
test_filtering.py ..........                                                                                                             [ 40%]
test_frame.py ....................................................................s....                                                  [ 56%]
test_frame_connexion.py ......                                                                                                           [ 57%]
test_frame_private.py ........................                                                                                           [ 62%]
test_geoFrame.py ...........................                                                                                             [ 68%]
test_geoSeries.py .......sss......s.s................s....                                                                               [ 76%]
test_indexing.py .....................                                                                                                   [ 80%]
test_internals.py .....                                                                                                                  [ 81%]
test_kmeans.py .........                                                                                                                 [ 83%]
test_naive_bayes.py .........                                                                                                            [ 85%]
test_series.py ...                                                                                                                       [ 86%]
test_sorting.py ............                                                                                                             [ 88%]
test_statistics.py ................................................                                                                      [ 98%]
test_utils.py ......                                                                                                                     [100%]
=========================================== 453 passed, 26 skipped, 17 warnings in 895.00s (0:14:55) ===========================================
```

### CODE REVIEW

**Comment by mrozpawel** [comment]
Fix for #53

File: nzpyida/__init__.py [hunk 1]
```diff
@@ -18,6 +18,8 @@
 from .base import IdaDataBase
 from .frame import IdaDataFrame
 from .series import IdaSeries
+from .geoFrame import IdaGeoDataFrame, IdaGeoSeries
+
 
 __all__ = ['sampledata', 'tests', 'aggregation', 
 		   'base', 'exceptions', 'filtering', 'frame', 'indexing',
```
Comment (mplpl) [inline nzpyida/__init__.py:25]: also add geoFrame and geoSeries here

**Review by mplpl — CHANGES_REQUESTED** [review]
We need to also modify MANIFEST.in file and remove global-exclude from it - otherwise getFrame and geoSeries will not be included in whl file after build.

**Comment by mplpl** [comment]
Can we also rename geoFrame.py to geo_frame.py and geoSeries.py to geo_series.py

File: docs/source/geospatial.rst [hunk 2]
```diff
@@ -39,12 +39,7 @@ The results are fetched and formatted into the corresponding data structure, for
 
 The following scenario illustrates how nzpyida works.
 
-Assuming that all ODBC connection parameters are correctly set, issue the following statements to connect to a
-database (in this case, a Netezza database with the name BLUDB) via ODBC:
-
-    >>> from nzpyida import IdaDataBase, IdaGeoDataFrame
-    >>> idadb = IdaDataBase('BLUDB')
-
+Assuming that you've got connection to the databased established via IdaDataBase object named 'idadb'.
```
Comment (mplpl) [inline docs/source/geospatial.rst:42]: Above this sentence, it talks about ODBC and JDBC - let's add nzpy there as well (and make it first option)

**Review by mplpl — APPROVED** [review]

**Pull request merged**